src/ncp/ncp_base_radio.cpp

```diff
@@ -452,21 +464,33 @@ otError NcpBase::HandlePropertySet_SPINEL_PROP_STREAM_RAW(uint8_t aHeader)
     otRadioFrame *frame;
 
     VerifyOrExit(otLinkRawIsEnabled(mInstance), error = OT_ERROR_INVALID_STATE);
+    if (otLinkRawIsTransmittingOrScanning(mInstance))
+    {
+#if OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE
+        ExitNow(error = EnqueuePendingCommand(kPendingCommandTypeTransmit, aHeader, 0));
```

Comment:
These Enqueue.. function seem to me like some additional feature. Consider additional #if to allow multipan without buffering feature.

Reply:
It would not work without this because when multiple hosts/stacks are operating, there will be situations where both the stacks would want to transmit.

Reply:
Unless platform can handle this case